filebeat/fileset/modules.go

```diff
@@ -48,22 +48,22 @@ func newModuleRegistry(modulesPath string,
 	beatInfo beat.Info,
 ) (*ModuleRegistry, error) {
 	reg := ModuleRegistry{
-		registry: map[string]map[string]*Fileset{},
+		registry: []map[string]map[string]*Fileset{},
 		log:      logp.NewLogger(logName),
 	}
-
 	for _, mcfg := range moduleConfigs {
-		if mcfg.Enabled != nil && !(*mcfg.Enabled) {
+		if mcfg.Module == "" || (mcfg.Enabled != nil && !(*mcfg.Enabled)) {
 			continue
 		}
-
+		logp.Info("Enabled " + mcfg.Module)
```

Comment:
The module registry has a logger, please use that instead of this global function call.

```suggestion
		reg.log.Info("Enabled " + mcfg.Module)
```

Reply:
This was left accidently as I was trying to follow the code logic.  I've removed it and updated the couple of other log lines to use the module registry logger.